Octokit.Reactive/Clients/IObservableOrganizationsClient.cs

```diff
@@ -16,6 +16,12 @@ public interface IObservableOrganizationsClient
         IObservableTeamsClient Team { get; }
 
         /// <summary>
+        /// A client for GitHub's Organization Hooks API.
+        /// </summary>
+        /// <remarks>See <a href="http://developer.github.com/v3/orgs/hooks/">Hooks API documentation</a> for more information.</remarks>
+        IObservableOrganizationHooksClient Hooks { get; }
```

Comment:
@ryangribble is the master of consistency around here, but I believe that (in a separate  PR) IObservableRepositoriesClient (and probably IRepositoriesClient) would need to be changed also